tools/database.py

```diff
@@ -44,26 +44,25 @@ def _db_exists(pathname: str) -> bool:
 def ensureDatabase(name: str, use_prefix: bool = True) -> None:
     "get sure that the db exists. If not, create one."
     realname = _db_route(name, prefix=use_prefix)
-    if not _db_exists(realname):
-        # the db does not exists, the tables must be created.
-        conn = sqlite3.connect(realname)
-        cur = conn.cursor()
-        cur.executescript("""
-CREATE TABLE Prompt (
-    id           INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT UNIQUE,
-    vendor_id    INTEGER,
-    product_id   INTEGER,
-    water_read   INTEGER,
-    cost         INTEGER,
-    time         TEXT
+    # if the db does not exists, the tables must be created.

```

Comment:
I removed the `if` statement, because `destroy-db.py` (and `Eliminar archivo SQLite.exe`) are destroying the database file, instead of a cleanup. It also could fail if an empty database is entered. So, removing that block avoids problems.

Also, it would be helpful to use `tools.database.ensureDatabase()` on `destroy-db.py`.